File: src/main/java/com/multicampus/gamesungcoding/a11ymarketserver/feature/seller/dto/SellerApplyRequest.java
package com.multicampus.gamesungcoding.a11ymarketserver.feature.seller.dto;

import jakarta.validation.constraints.NotBlank;

public record SellerApplyRequest(
        @NotBlank String sellerName,
        @NotBlank String businessNumber,
        String sellerIntro) {
}

File: src/main/kotlin/com/multicampus/gamesungcoding/a11ymarketserver/feature/seller/dto/SellerApplyRequest.kt
package com.multicampus.gamesungcoding.a11ymarketserver.feature.seller.dto

import jakarta.validation.constraints.NotBlank

data class SellerApplyRequest(
    @field:NotBlank val sellerName: String,
    @field:NotBlank val businessNumber: String,
    val sellerIntro: String? = null
)
